CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Code of Conduct
```

Comment:
Why is this file the same as the Code of Conduct?

Reply:
Since you opened an issue asking to add code of conduct

Reply:
But this filename is CONTRIBUTING.md 
You created 2 files CONTRIBUTING.md and CODE_OF_CONDUCT.md both with the same text inside. 

The CODE_OF_CONDUCT.md file looks great (except it needs you to fill in the proper email). Whats the purpose of the CONTRIBUTING.md file? Why does it have the same text?

Reply:
Nvm I fixed it